app/services/katello/candlepin/consumer.rb

```diff
@@ -132,11 +132,18 @@ def compliance_reasons
 
       def self.orphaned_consumer_ids
         #returns consumer ids in candlepin with no matching katello entry
-        cp_consumers = ::Katello::Resources::Candlepin::Consumer.get({})
-        cp_consumers.reject! { |consumer| consumer['type']['label'] == 'uebercert' }
-        cp_consumer_ids = cp_consumers.map { |cons| cons["uuid"] }
-        katello_consumer_ids = ::Katello::Host::SubscriptionFacet.pluck(:uuid)
-        cp_consumer_ids - katello_consumer_ids
+        orphaned_ids = []
+        User.as_anonymous_admin do
+          cp_consumers = []
+          Organization.all.each do |org|
+            cp_consumers += ::Katello::Resources::Candlepin::Consumer.get('owner' => org.label)
+          end
```

Comment:
yeah, because the get() returns an array, so doing collect, you end up with an array of arrays:

```
> [1,2,3].collect{|i| [i]}
 => [[1], [2], [3]] 
```